.gitlab-ci.yml

```diff
@@ -3,72 +3,66 @@ variables:
   GIT_SUBMODULE_STRATEGY: none
 
 stages:
-  - prereq
-  - check
-  - test-short
-  - test-long
+  - build
+  - test
 
-build-toolchain:
-  when: manual
-  stage: prereq
+build-repos:
+  stage: build
   tags:
     - bsg
-  before_script:
+  script:
+    - echo "Cloning repos..."
     - rm -rf *
-    - echo "Cloning Manycore"
-    - git clone -b $CI_COMMIT_REF_NAME https://github.com/bespoke-silicon-group/bsg_manycore.git
-    - echo "Copying cadenv from local directory"
     - cp -r $BSG_CADENV_DIR bsg_cadenv
-  script:
-    - echo "Building Manycore Toolchain."
-    - cd bsg_manycore
-    - make checkout_submodules
-    - make -C software/riscv-tools clean-all install-clean
+    - git clone https://github.com/bespoke-silicon-group/basejump_stl.git
+    - git clone --recursive -b $CI_COMMIT_REF_NAME https://github.com/bespoke-silicon-group/bsg_manycore.git
+    - git clone https://github.com/bespoke-silicon-group/bsg_f1.git 
+    - echo "Building toolchain"
+    - cd bsg_manycore/software/riscv-tools
+    - make clean-all install-clean
   cache:
     key: $CI_COMMIT_REF_SLUG
     paths:
-      - $CI_PROJECT_DIR/bsg_manycore/imports/
-      - $CI_PROJECT_DIR/bsg_manycore/software/riscv-tools/
+      - $CI_PROJECT_DIR/bsg_cadenv
+      - $CI_PROJECT_DIR/basejump_stl
+      - $CI_PROJECT_DIR/bsg_manycore
```

Comment:
This means that you need to rebuild the toolchain on every regression.  Is that what you want?